### ISSUE TITLE
Problemas con Windows 11 y elección de navegador web

### ISSUE THREAD

**Issue by chachak** [DanysysTeam/AutoIt-Scripts#2]
Enhorabuena por tu código, Dany.

Estos dos ejemplos funcionan bien con Windows 10, pero no bajo Windows 11:

_Set_PTA("FirefoxURL", "http")
_Set_FTA("FirefoxHTML", ".html")

**Comment by Danyfirex** [comment]
Para las extensiones te funciona en Windows 11?

**Comment by chachak** [comment]
Sí, funciona, lo he probado con imágenes y archivos multimedia.

Para archivos .pdf, tengo mejores resultados últimamente con el ProgID Acrobat.Document.DC que con AcroExch.Document.DC (en ambos casos, SFTA me asocia correctamente los .pdf con Acrobat Reader **DC**, de 32 y 64 bits). Tanto en Windows 10, como en Windows 11. No tiene demasiada lógica, pero...

**Comment by Danyfirex** [comment]
Recuerdo que con algunas pruebas que hice hace tiempo me pasaba algo similar y creo que es porque no ha sido colocado ningún protocolo por defecto antes. Prueba a colocar algún navegador por defecto manualmente y luego usa SFTA para estar seguro si es el mismo error. Es algo raro que me paso en algunas pruebas pero no le preste mucha atención, así que tendré que revisar profundamente.

**Comment by chachak** [comment]
No creo que tenga algo que ver que no haya un protocolo por defecto, o quizás se me escapa algo. En Windows 10, igual te ayuda esto:

RegDelete ('HKEY_CLASSES_ROOT\FirefoxHTML\ShellEx')
RegDelete ('HKEY_CLASSES_ROOT\htmlfile\shellex')

Es para que, después de cambiar la asociación, los archivos tengan el icono correcto. Se me ocurrió después de leer algo sobre iconhandlers (no entendí casi nada, pero me sirvió de punto de partida).

**Comment by Danyfirex** [comment]
No he encontrado ningún error en mis pruebas todo funciona correctamente, dejare el issue abierto por si alguien más tiene un error parecido y puede aportar algo más.
